ts/components/PinCreationForm.tsx

```diff
@@ -194,6 +219,16 @@ export const PinCreationForm = ({ onSubmit }: Props) => {
         </View>
       </ScrollView>
 
+      {isDevEnv && (
+        <View style={{ alignSelf: "center" }}>
+          <ButtonOutline
+            label={"Enter Pin: 162534 (DevEnv Only)"}
```

Comment:
If we put `162534` in a shared constant I suggest to use string interpolation here.